docs/radio.rst

```diff
@@ -132,6 +133,27 @@ Functions
 
     A ``ValueError`` exception is raised if conversion to string fails.
 
+.. py:function:: receive_all()
+
+    Returns a tuple containing four values representing the next incoming
+    message on the message queue. If there no pending messages then ``None``
+    is returned.
+
+    The four values in the tuple represent:
+
+    * the next incoming message on the message queue as bytes.
+    * the RSSI signal strength: a value between 0 (strongest) and 255 (weakest).
```

Comment:
I suspect the RSSI on this system is in dBm (rather than dB) but I can not find notes anywhere confirming this.
I see on the Lancaster runtime uBit.radio that the Transmission Power has eight user configurable settings from 0 (-30dbm) to 7 (+4dbm).